srcpkgs/clex/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'clex'
+pkgname=clex
+version=4.6.patch9
+revision=1
+archs="x86_64 i868 x86_64-musl"
+build_style="gnu-configure"
+configure_args="--prefix=/usr --sysconfdir=/etc --infodir=/usr/share/info --mandir=/usr/share/man --localstatedir=/var"
+hostmakedepends="ncurses-devel ncurses"
```

Comment:
This is most likely wrong and should instead go into makedepends. See Manual.md for difference between hostmakedepends and makedepends.